backend/schedule/admin.py

```diff
@@ -25,9 +28,85 @@
     ScheduleItemAttendee,
     ScheduleItemInvitation,
     Slot,
+    SpeakerVoucher,
 )
 
 
+@admin.action(description="Generate voucher codes")
+def generate_voucher_codes(modeladmin, request, queryset):
```

Comment:
We need to refactor the admins, they have a lot of logic... we should move in some service level this stuff...